src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -49,13 +49,22 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		// Buttons in the range [0.0, 1.0]
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
+		[iOS (12,2), Mac (10,14,4, onlyOn64: true), TV (12,2)]
+		public bool supportsClickableThumbsticks;
```

Comment:
Public API shouldn't start with a lower-cased letter: `SupportsClickableThumbsticks`.

Same for the to other fields just below.

Reply:
Missed this one too, thanks!